BUILD.md

```diff
@@ -9,103 +9,136 @@ See also README.md Dependencies section
+```
 sudo apt-get install libgoogle-glog-dev libboost-system-dev \
-     libdouble-conversion-dev # and optionally: libjemalloc-dev
-
-# OR
-
-# If double-conversion isn't available:
-
+libdouble-conversion-dev libjemalloc-dev
+```
+*If double-conversion isn't available via apt-get:*
+```
 git clone https://github.com/floitsch/double-conversion.git
 cd double-conversion; cmake . ; make -j && sudo make install
+```
 
+__Otherwise, Build gflags and glog from source__
 
-# Get and install gflags and glog - it's important to build and configure
-# gflags correctly for glog to pick it up and avoid linking errors later
-# or getting a wdt without flags working
+*It's important to build and configure gflags correctly for glog to pick it up
+and avoid linking errors later or getting a wdt without flags working*
+```
 git clone https://github.com/schuhschuh/gflags.git
 mkdir gflags/build
 cd gflags/build
 cmake -D GFLAGS_NAMESPACE=google -D BUILD_SHARED_LIBS=on ..
 make -j && sudo make install
-# Glog
+```
+
+```
 svn checkout http://google-glog.googlecode.com/svn/trunk/ glog
 cd glog
 ./configure # add --with-gflags=whereyouinstalledgflags
 # to avoid ERROR: unknown command line flag 'minloglevel' later
 make -j && sudo make install
+```
 
 
 
 
-# And finally Build wdt itself:
+__Build wdt from source__
+```
 cmake pathtowdtsrcdir -DBUILD_TESTING=on  # skip -D... if you don't want tests
 make -j
 make test
 sudo make install
+```
+# Install on Mac OS X
 
-# Macos
-
-Install Xcode 6 and the command line tools (so
+__Install Xcode 6 and the command line tools__ (so
 /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++
 exists)
 
-## cmake
-# Until 3.3 is out we need the latest on a Mac to
-# get CXX 11 support: ( http://www.cmake.org/Bug/view.php?id=15355 )
+__Install Cmake__
 
+```
 git clone http://cmake.org/cmake.git
 cd cmake
 ./bootstrap --prefix=/usr --parallel=16
 make -j && sudo make install
+cd ../
+```
+__Install homebrew (http://brew.sh/)__
 
-## Get homebrew at http://brew.sh/
-
-### Glog and gflags and boost
+__Install glog and gflags and boost__
+```
 brew update
 brew install glog gflags boost
-## Double conversion
+```
+__Install Double conversion__
+```
 git clone https://github.com/floitsch/double-conversion.git
-cd double-conversion; cmake . ; make -j && sudo make install
-
-### Get folly source tree
-As above
-
-## finally build wdt itself:
-mkdir wdt_mac
-cd wdt_mac
-# For Xcode:
+cd double-conversion; cmake . ; make -j && sudo make install;
+cd ../
+```
+__Build wdt from source__
+```
+mkdir wdt-mac
+cd wdt-mac
+```
+Get folly source for use during build
+```
+git clone https://github.com/facebook/folly.git
+```
+Fetch wdt source
+```
+git clone https://github.com/facebook/wdt.git
+```
+Using Xcode:
+```
 cmake pathtosrccode -G Xcode -DBUILD_TESTING=on
-# Using unix makefiles
+```
+
+Using Unix makefiles:
+```
 cmake pathtosrccode -G "Unix Makefiles" -DBUILD_TESTING=on
 make -j
 make test
```

Comment:
same here, 
CTEST_OUTPUT_ON_FAILURE=1 make test
and add
Until Issue #30 is fixed:
WDT_TEST_SYMLINKS=0 CTEST_OUTPUT_ON_FAILURE=1 make test